app/config/FrontendAppConfig.scala

```diff
@@ -227,5 +215,6 @@ class FrontendAppConfig @Inject()(runModeConfiguration: Configuration, environme
   lazy val pspSchemeDashboardUrl: String = loadConfig("urls.pspSchemeDashboard")
 
   lazy val hideAftTile: Boolean = runModeConfiguration.get[Boolean]("hideAftTile")
-
+  lazy val interimDashboard: Boolean = runModeConfiguration.get[Boolean]("interim-dashboard")
```

Comment:
Don't need interim-dashboard. We've deployed this now.

Reply:
done